packages/devtools/lib/src/ui/plotly.dart

```diff
@@ -0,0 +1,237 @@
+@JS()
+library plotly;
+
+import 'package:js/js.dart';
+
+@JS()
+external void hookupPlotlyClick(String domName, Function jsFunction);
```

Comment:
consider making this

@JS('hookupPlotlyClick')
external void _hookupPlotlyClick(String domName, Function jsFunction);
as I don't think you intend users to call it directly.

Reply:
Done.